pkg/version/version.go

```diff
@@ -0,0 +1,26 @@
+package version
+
+import (
+	"errors"
+	"fmt"
+	"runtime/debug"
+)
+
+// version returns the version of the main module
+func version() (string, error) {
+	info, ok := debug.ReadBuildInfo()
+	if !ok {
+		return "", errors.New("binary has not been built with module support")
+	}
+	return info.Main.Version, nil
+}
+
+// Print prints the main module version on stdout
+func Print() error {
+	v, err := version()
+	if err != nil {
+		return err
```

Comment:
Looks like nothing will get printed for the end-user. We should probably print `Version: Unknown` in this case.